api/v1beta1/machine_types.go

```diff
@@ -200,22 +200,70 @@ const (
 	// Note: this could happen when creating the machine. However, this state should be treated as an error if it lasts indefinitely.
 	MachineNodeDoesNotExistV1Beta2Reason = ObjectDoesNotExistV1Beta2Reason
 
-	// MachineNodeDeletedV1Beta2Reason surfaces when  the node hosted on the machine has been deleted.
+	// MachineNodeDeletedV1Beta2Reason surfaces when the node hosted on the machine has been deleted.
 	// Note: controllers can't identify if the Node was deleted by the controller itself, e.g.
 	// during the deletion workflow, or by a users.
 	MachineNodeDeletedV1Beta2Reason = ObjectDeletedV1Beta2Reason
 )
 
-// Machine's HealthCheckSucceeded and OwnerRemediated conditions and corresponding reasons that will be used in v1Beta2 API version.
-// Note: HealthCheckSucceeded and OwnerRemediated condition are set by the MachineHealthCheck controller.
+// Machine's HealthCheckSucceeded condition and corresponding reasons that will be used in v1Beta2 API version.
+// Note: HealthCheckSucceeded condition ia set by the MachineHealthCheck controller.
 const (
 	// MachineHealthCheckSucceededV1Beta2Condition is true if MHC instances targeting this machine report the Machine
 	// is healthy according to the definition of healthy present in the spec of the MachineHealthCheck object.
 	MachineHealthCheckSucceededV1Beta2Condition = "HealthCheckSucceeded"
 
+	// MachineHealthCheckPassedV1Beta2Reason surfaces when a machine passes all the health checks defined by a MachineHealthCheck object.
+	MachineHealthCheckPassedV1Beta2Reason = "HealthCheckPassed"
```

Comment:
```suggestion
	MachineHealthCheckSucceededV1Beta2Reason = "HealthCheckSucceeded"
```
Except if we see a difference between succeeded and passed?

IIRC succeeded is what we usually use

Reply:
Fixed, but I plan to make a round of refinements on those conditions